### DISCUSSION TITLE
Adjustment to Half Time/Daycore multiplier scaling

### DISCUSSION THREAD

**Discussion by Non-Binary-Programmer** [ppy/osu#23245] [Ideas]
I think that Half Time and Daycore should scale with every 5% rate adjustment instead of every 10%. This makes sense for two main reasons. First of all, this would create better parity with Double Time, which currently has 10 score steps while Half Time has 5. Second, the default value for Half Time, and the one that the legacy player base is most familiar with, is 0.75x. However, the current multiplier scaling makes this value suboptimal for competitive score play, as 0.75x speed currently has the same multiplier as the easier 0.7x speed. This would not be a problem if Half Time scaled with every 5%, as then 0.75x speed would be the slowest speed within its score bracket.

**Comment by Rekunan** [comment]
I think we should also do this for DT/NC. Having increment parity is a higher priority than score step parity, as it's more intuitive.

**Comment by respektive** [comment]
I agree with this. I think its very weird that the default rate for HT is basically a sub-optimal one for competitive play.

**Comment by zzirSupeR** [comment]
Would this mean that with HT/DC if the speed is set to 0.99x then the multiplier can be like 0.95x? I've been thinking about this for a little while. I just like how DC lowers the pitch of the song but not the difficulty of the map (if speed is set to 0.99x). Maybe we can do a poll? Maybe we can add a pitch shifter mod? That would be cool.
